specialisation/homework1.py

```diff
@@ -0,0 +1,38 @@
+import abc
+class Shape(object):
+    __metaclass__ = abc.ABCMeta\
+
+    @abc.abstractmethod
+    def calc_perimeter(self, input):
+        """Method documentation"""
+        return
+
+class Triangle(Shape):
+    def __init__(self, a, b, c):
+        self.a = a
+        self.b = b
+        self.c = 
+    def calc_perimeter(self):
+        return self.a + self.b + self.c
+
+class Rectangle(Shape):
+    def __init__(self, a, b):
+        self.a = a
+        self.b = b
+    def calc_perimeter(self):
+        return 2 * self.a + self.b
```

Comment:
Calculation of perimeter is incorrect. The length and breadth should be multiplied by 2. i.e 
`2 * (self.a + self.b)`